### PR TITLE
Merge the two contributing docs and create instructions for updating an op

### PR DESCRIPTION
**Pull request by justinchuby**
### Description

Merge the two contributing docs and create instructions for updating an op

### Motivation and Context

Fixes https://github.com/onnx/onnx/issues/5379

### CODE REVIEW

**Review by jcwchen — APPROVED** [review]
LGTM. Thanks for merging them into once place!

File: docs/AddNewOp.md [hunk 1]
```diff
@@ -85,21 +89,31 @@ rank inference at the very least (adding right amount of dimensions to the outpu
...
 Once the PR is reviewed and signed off by the Operators SIG, it will be merged. Your new operator/function will be part of the main branch and available to anyone building from source. These are not official releases. ONNX periodically releases official new versions that are a snapshot of the main branch. Your new operator/function will be part of that release.
 
-## Removing operator or function <a name="removing_operator_or_function"></a>
+## Updating an existing operator
+
+The definition of an existing operator may need to be updated when e.g. there are new scenarios or input types to support. The process is largely similar to that for creating a new operator.
+
+### Checklist
+
+Use this checklist when updating an existing operator: https://github.com/onnx/onnx/wiki/Checklist-for-updating-an-existing-operator
```
Comment (gramalingam) [inline docs/AddNewOp.md:110]: `Add the operator to "onnx/defs/operator_sets.h"` should be `Add the operator to "onnx/defs/operator_sets.h" or "onnx/defs/operator_sets_ml.h"` in the checklist

Comment (gramalingam) [reply docs/AddNewOp.md:110]: Would be helpful if the checklist mentions the use of auto-update-doc label

Comment (justinchuby) [reply docs/AddNewOp.md:110]: Done

**Review by gramalingam — APPROVED** [review]

**Pull request merged**